packages/ripple-binary-codec/src/enums/definitions-xahau.json

```diff
@@ -0,0 +1,2740 @@
+{
```

Comment:
This should go in the `test` folder, not the `src` folder. And perhaps should be renamed to something like `definitions-sample.json` so that people don't think it's the current production file.

This is because this file would increase the size of the package by about 6%, and it's therefore not sustainable to do for every XRPL-based chain. I'd recommend maintaining this file in a separate repo and pointing people towards that.

Reply:
Will do in separate repo 👍

Reply:
The rest of the PR was fine, just this one file was what I suggested you move to a new repo 🙂

Reply:
@mvadari Done, loading JSON from external dev-dependency.